Changes in upstream/master
	Modified   copyright-format/copyright-format-1.0.xml
diff --git a/copyright-format/copyright-format-1.0.xml b/copyright-format/copyright-format-1.0.xml
index 217e8dd..1cbe0d7 100644
--- a/copyright-format/copyright-format-1.0.xml
+++ b/copyright-format/copyright-format-1.0.xml
@@ -240,4 +240,10 @@
         <listitem>
           <para>
+            <link linkend="license-grant-field">License-Grant</link>:
+            optional.
+          </para>
+        </listitem>
+        <listitem>
+          <para>
             <link linkend="copyright-field">Copyright</link>: optional.
           </para>
@@ -245,16 +251,25 @@
       </itemizedlist>
       <para>
-        The <varname>Copyright</varname> and <varname>License</varname>
-        fields in the <emphasis>header paragraph</emphasis> may complement
-        but do not replace the <emphasis>Files paragraphs</emphasis>.  If
-        present, they summarise the copyright notices or redistribution
-        terms for the package as a whole.  For example, when a work
-        combines a permissive and a copyleft license,
-        <varname>License</varname> can be used to clarify the license
-        terms for the combination.  <varname>Copyright</varname> and
-        <varname>License</varname> together can also be used to document a
-        <emphasis>compilation copyright</emphasis> and license.  It is
-        possible to use only <varname>License</varname> in the header
-        paragraph, but <varname>Copyright</varname> alone makes no sense.
+        The <varname>Copyright</varname>, <varname>License-Grant</varname>,
+        and <varname>License</varname> fields in the <emphasis>header
+        paragraph</emphasis> may complement but do not replace the fields
+        in the <emphasis>Files paragraphs</emphasis>.
+        If present, they summarise the copyright notices, grant of license,
+        and/or redistribution terms for the package as a whole.
+      </para>
+      <para>
+        For example, when a work has a grant of license under both a
+        permissive and a copyleft license, <varname>License-Grant</varname>
+        and <varname>License</varname> can be used to clarify the license
+        terms for the combination. <varname>Copyright</varname> and
+        <varname>License</varname> together can also be used to
+        document a <emphasis>compilation copyright</emphasis> and
+        license.
+      </para>
+      <para>
+        It is valid to use <varname>License-Grant</varname> and/or
+        <varname>License</varname> in the header paragraph without an
+        accompanying <varname>Copyright</varname> field, but
+        <varname>Copyright</varname> alone is not sufficient.
       </para>
 
@@ -295,4 +310,10 @@ Source: http://www.example.com/software/project</programlisting>
         <listitem>
           <para>
+            <link linkend="license-grant-field">License-Grant</link>:
+            optional.
+          </para>
+        </listitem>
+        <listitem>
+          <para>
             <link linkend="license-field">License</link>: required.
           </para>
@@ -309,31 +330,54 @@ Source: http://www.example.com/software/project</programlisting>
 <programlisting>Files: *
 Copyright: 1975-2010 Ulla Upstream
+License-Grant:
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
 License: GPL-2+
 
 Files: debian/*
 Copyright: 2010 Daniela Debianizer
+License-Grant:
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
 License: GPL-2+
 
 Files: debian/patches/fancy-feature
 Copyright: 2010 Daniela Debianizer
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
 License: GPL-3+
 
 Files: */*.1
 Copyright: 2010 Manuela Manpager
-License: GPL-2+</programlisting>
+License-Grant:
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+License: GPL-2+
+</programlisting>
         <para>
-          In this example, all files are copyright by the upstream and licensed
-          under the GPL, version 2 or later, with three exceptions.  All the
-          Debian packaging files are copyright by the packager, and further one
-          specific file providing a new feature is licensed differently.
-          Finally, there are some manual pages added to the package, written by
-          a third person.
+          In this example, all files are copyright by the upstream and
+          that copyright holder grants license under the GPL, version
+          2 or later. There are three exceptions. All the Debian
+          packaging files have copyright held by the packager, and
+          further one specific file providing a new feature has a
+          different grant of license. Finally, there are some manual
+          pages added to the package, with copyright held by a third
+          person.
         </para>
         <para>
-          Since the license of the manual pages is the same as the other
-          files in the package, the last paragraph above could instead be
-          combined with the first paragraph, listing both copyright
-          statements in one <varname>Copyright</varname> field.  Whether
-          to combine paragraphs with the same license is left to the
+          Since the <varname>License-Grant</varname> of the manual
+          pages is the same as most other files in the package, the
+          final paragraph above could instead be combined with the
+          first paragraph, listing both copyright statements in one
+          <varname>Copyright</varname> field. Whether to combine
+          paragraphs with the same grant of license is left to the
           discretion of the author of the
           <filename>debian/copyright</filename> file.
@@ -345,10 +389,16 @@ License: GPL-2+</programlisting>
       <title>Stand-alone License Paragraph (optional, repeatable)</title>
       <para>
-        Stand-alone <varname>License</varname> paragraphs can be used to
-        provide the full license text for a given license once, instead of
-        repeating it in each <varname>Files</varname> paragraph that refers to
-        it.  The first line of the <varname>License</varname> field must be a
-        single license short name or a short name followed by a license
-        exception.
+        Stand-alone <varname>License</varname> paragraphs can be used
+        to provide the full license text for a given license once,
+        instead of repeating it in each <varname>Files</varname>
+        paragraph that refers to it. The
+        <varname>License-Grant</varname> field can be used to clarify
+        exactly what text from the copyright holder grants the
+        particular set of license conditions.
+      </para>
+      <para>
+        The synopsis (first line) of the <varname>License</varname>
+        field must be a single license short name or a short name
+        followed by a license exception.
       </para>
       <para>
@@ -374,4 +424,9 @@ License: GPL-2+</programlisting>
 Copyright: 1993, John Doe
            1993, Joe Average
+License-Grant:
+ This is free software: you may modify and/or redistribute under your
+ choice of the Mozilla Public License 1.1, the GNU General Public
+ License version 2, or the GNU Lesser General Public License version
+ 2.1.
 License: MPL-1.1 or GPL-2 or LGPL-2.1
 
@@ -391,8 +446,15 @@ License: LGPL-2.1
 Copyright: 1993, John Doe
            1993, Joe Average
+License-Grant:
+ Permission to redistribute is granted under Mozilla Public License
+ version 1.1.
 License: MPL-1.1
 
 Files: src/js/fdlibm/*
 Copyright: 1993, J-Random Corporation
+License-Grant:
+ J-Random Corporation hereby grants all recipients permission to
+ change, run, or redistribute this work under the terms of the Mozilla
+ Public License version 1.1.
 License: MPL-1.1
 
@@ -471,38 +533,71 @@ License: MPL-1.1
     </section>
 
+    <section id="license-grant-field">
+      <title><varname>License-Grant</varname></title>
+      <para>
+        Formatted text, no synopsis. This field contains the verbatim
+        text from the copyright holder that grants license explicitly
+        to the recipient of the work under specific conditions. This
+        explicit grant, associated with a specific work, clarifies the
+        copyright holder's explicit intent to grant license in this
+        work to the recipient under specific terms.
+      </para>
+      <para>
+        The full license terms and conditions themselves – the license
+        text – belongs in a
+        <link linkend="license-field">License</link> field.
+      </para>
+      <para>
+        In the <link linkend="header-paragraph">header
+        paragraph</link>, this field gives the grant of license from
+        the copyright holder of the work as a whole. In a <link
+        linkend="files-paragraph">Files paragraph</link>, this field
+        gives the grant of license for the files listed in the
+        <varname>Files</varname> field for this paragraph.
+      </para>
+    </section>
+
     <section id="license-field">
       <title><varname>License</varname></title>
       <para>
-        Formatted text, with synopsis.  In the header paragraph, this field
-        gives the license information for the package as a whole, which may
-        be different or simplified from a combination of all the per-file
-        license information.  In a Files paragraph, this field gives the
-        licensing terms for the files listed in the <varname>Files</varname>
-        field for this paragraph.  In a stand-alone License paragraph, it
-        gives the licensing terms for those paragraphs which reference it.
+        Formatted text, with synopsis.
+      </para>
+      <para>
+        In the <link linkend="header-paragraph">header
+        paragraph</link>, this field gives the license information for
+        the package as a whole, which may be different or simplified
+        from a combination of all the per-file license information. In
+        a <link linkend="files-paragraph">Files paragraph</link>, this
+        field gives the licensing terms for the files listed in the
+        <varname>Files</varname> field for this paragraph. In a <link
+        linkend="stand-alone-license-paragraph">stand-alone License
+        paragraph</link>, it gives the licensing terms for those
+        paragraphs which reference it.
       </para>
       <para>
-        First line: an abbreviated name for the license, or expression
-        giving alternatives (see the <link linkend="license-short-name">Short
-        name</link> section for a list of standard abbreviations).  If
-        there are licenses present in the package without a standard short
-        name, an arbitrary short name may be assigned for these licenses. 
-        These arbitrary names are only guaranteed to be unique within a
-        single copyright file.
+        First line (synopsis): an abbreviated name for the license, or
+        expression giving alternatives (see the <link
+        linkend="license-short-name">Short name</link> section for a
+        list of standard abbreviations). If there are licenses present
+        in the package without a standard short name, an arbitrary
+        short name may be assigned for these licenses. These arbitrary
+        names are only guaranteed to be unique within a single
+        copyright file.
       </para>
       <para>
         If there are no remaining lines, then all of the short names
-        or short names followed by license exceptions making up the
-        first line must be described in <link
+        or short names followed by license exceptions in the synopsis
+        must be described in <link
         linkend="stand-alone-license-paragraph">stand-alone License
-        paragraphs</link>.  Otherwise, this field should either
-        include the full text of the license(s) or include a pointer to the
-        license file under <filename>/usr/share/common-licenses</filename>. 
-        This field should include all text needed in order to fulfill both
-        Debian Policy's requirement for including a copy of the software's
+        paragraphs</link>. Otherwise, this field should either include
+        the full text of the license(s) or include a pointer to the
+        license file under
+        <filename>/usr/share/common-licenses</filename>. This field
+        should include all text needed in order to fulfill both Debian
+        Policy's requirement for including a copy of the software's
         distribution license (<ulink
         url="http://www.debian.org/doc/debian-policy/ch-docs#s-copyrightfile">12.5</ulink>),
-        and any license requirements to include warranty disclaimers or
-        other notices with the binary package.
+        and any license requirements to include warranty disclaimers
+        or other notices with the binary package.
       </para>
     </section>
@@ -644,5 +739,5 @@ Copyright 2009, 2010 Angela Watts</programlisting>
         able to correlate the licenses of multiple pieces of software.  To that
         end, this spec defines standard short names for a number of commonly
-        used licenses, which can be used in the first line of a
+        used licenses, which can be used in the synopsis (first line) of a
         <varname>License</varname> field.
       </para>

